internal/gatewayapi/backendtlspolicy.go

```diff
@@ -19,6 +21,8 @@ import (
 	"github.com/envoyproxy/gateway/internal/ir"
 )
 
+var ErrBackendTLSPolicyInvalidKind = fmt.Errorf("no ca found in referred ConfigMap, Secret or ClusterTrustBundle")
```

Comment:
this is unrelated to this PR, but can be update the string to - "no CA bundle found in referenced ConfigMap, Secret, or ClusterTrustBundle"